modules/@apostrophecms/piece-type/index.js

```diff
@@ -683,6 +683,7 @@ module.exports = {
         browserOptions.contextual = self.contextual;
         browserOptions.batchOperations = self.batchOperations;
         browserOptions.insertViaUpload = self.options.insertViaUpload;
+        browserOptions.quickCreate = true;
```

Comment:
I was thinking more like actually setting:

```
options: {
  quickCreate: true
}
```

And here where you build browserOptions, pulling that in as self.options.quickCreate.

Similarly global can shut it off cleanly via configuration.

Reply:
This way devs can opt out for piece types that are not so primary.